cocos/scene-graph/component-event-handler.ts

```diff
@@ -174,18 +174,18 @@ export class EventHandler {
         handler.apply(comp, params);
     }
 
-    private _compName2Id (compName) {
+    private _compName2Id (compName): any {
```

Comment:
Return string?

Reply:
here we use legacyCC in this implementation, so TS can only infer the return type as `any`